ovs/extensions/healthcheck/volumedriver/volumedriver_health_check.py

```diff
@@ -127,56 +139,232 @@ def check_volumedrivers(logger):
         if len(vpools) != 0:
             for vp in vpools:
                 name = "ovs-healthcheck-test-{0}.raw".format(VolumedriverHealthCheck.MACHINE_ID)
-                if vp.guid in VolumedriverHealthCheck.MACHINE_DETAILS.vpools_guids:
-                    try:
-                        # delete if previous vdisk with this name exists
-                        if VolumedriverHealthCheck._check_volumedriver_remove(vpool_name=vp.name, vdisk_name=name,
-                                                                              present=False):
-                            storagedriver_guid = next((storagedriver.guid for storagedriver in vp.storagedrivers
-                                                       if storagedriver.storagedriver_id == vp.name +
-                                                       VolumedriverHealthCheck.MACHINE_ID))
-                            # create a new one
-                            if VolumedriverHealthCheck._check_volumedriver(name, storagedriver_guid, vp.name):
-                                # delete the recently created
-                                if VolumedriverHealthCheck._check_volumedriver_remove(vpool_name=vp.name,
-                                                                                      vdisk_name=name):
-                                    # working
-                                    logger.success("Volumedriver of vPool '{0}' is working fine!".format(vp.name),
-                                                   'volumedriver_{0}'.format(vp.name))
+                with volatile_mutex('ovs-healthcheck_check-volumedrivers_{0}'
```

Comment:
This test only tests the local volumedriver. Don't lock it for the whole cluster.